commands/build_stop.js

```diff
@@ -25,13 +25,34 @@ function stop_build(args) {
     } else {
       reject("Access Key not provided");
     }
-
-    if (
-      !("buildId" in args) ||
-      args["buildid"] == "" ||
-      args["buildid"] == undefined
-    ) {
-      reject("Please provide a build ID");
+    if ("stop_last_session" in args) {
+      const file_path = "lambdatest_run.json";
+      if (fs.existsSync(file_path)) {
+        let lambda_run = fs.readFileSync(file_path);
+        try {
+          let lambda_run_obj = JSON.parse(lambda_run);
+          if (!("session_id" in lambda_run_obj)) {
+            throw new Error("session_id is missing from the file");
+          }
+          args.session_id = lambda_run_obj.session_id;
+        } catch (e) {
+          reject(
+            "Error!! lambdatest_run.json file is tempered Err: " + e.message
```

Comment:
@japneetlambdatest `tampered`.
Apologies for being english nazi 😄

Reply:
Done